### ISSUE TITLE
after latest updates Qt and Ansible docsets are broken

### ISSUE THREAD

**Issue by zetok** [zealdocs/zeal#652]
Ansible one just slightly, it should open "first" page in docs, not the modules page

Qt one is completely broken, not usable.

**Comment by trollixx** [comment]
For the Ansible docset its index page is defined as `docs.ansible.com/ansible/modules_by_category.html`, no idea why not `docs.ansible.com/ansible/index.html`, but that's a question for @Kapeli.

What's wrong with the Qt docset, and which one exactly, Qt 4 or Qt 5?

**Comment by zetok** [comment]
Qt 5, lots of stuff under `Modules` doesn't open, e.g. `QtPrintSupport`.

There is also a breakage in rendering of Qt docs, before they had working CSS and codeblocks, now they don't.

I'm not sure about this, but it seems like lots of links in Qt docs now point to online version, rather than the offline docs.

**Comment by trollixx** [comment]
Okay, looks like everything has nothing to do with Zeal, but rather docset issues. Leaving open for Bogdan to see.

**Comment by Kapeli** [comment]
`QtPrintSupport` works fine in Dash as far as I can tell. Also code blocks.

![screen shot 2016-11-15 at 22 21 00](https://cloud.githubusercontent.com/assets/1445635/20322456/de2b2f66-ab81-11e6-95aa-1797f2fb9c85.png)

**Comment by trollixx** [comment]
I don't see any issues with the Qt 5 docset either. @zetok please give some concrete examples of broken links.

**Comment by zetok** [comment]
![spectacle cn1704](https://cloud.githubusercontent.com/assets/3148759/20323399/74bc3c8a-ab74-11e6-9f52-65e738f93fa9.png)
↑ just some broken CSS

↓ when trying to switch to some other module nothing happens
![spectacle gx1704](https://cloud.githubusercontent.com/assets/3148759/20323429/900d6194-ab74-11e6-9743-1d89740ee48c.png)

Given that you couldn't reproduce, I've figured out what the issue with Qt docs is. I've removed them, and downloaded again, that fixed the issue.

With that being said, it would be really nice if zeal actually verified whether downloaded/installed docs are actually fully installed/downloaded.
